src/style.css

```diff
@@ -0,0 +1,118 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  font-style: normal;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 600;
+  font-style: normal;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.searchbar {
+  margin: 20px auto 0;
+}
+
+.big-searchbar {
+  width: 100%;
+  height: 70px;
+}
+
+.small-searchbar {
+  width: 100%;
```

Comment:
don't repeat yourself
you could use one separate class and use set common styles there (`width: 100%;`)
upd you already have such class (`searchbar`)

Reply:
check all such cases

Reply:
Thank you for your feedback. Styles fixed.